UITests/Helper/Receipts/ReceiptsList.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+class ReceiptsList {
```

Comment:
Change class name to ``ListReceiptTest``` and change the file name to ```ListReceiptTests.swift```

Reply:
fixed